### ISSUE TITLE
Problem when formatting the Markdown in Spanish

### ISSUE THREAD

**Issue by humitos** [DjangoGirls/tutorial#722]
The dotted list is not working properly in the Spanish version. Please check the differences:
- http://tutorial.djangogirls.org/en/python_introduction/index.html#your-first-python-command
- http://tutorial.djangogirls.org/es/python_introduction/index.html#tu-primer-comando-en-python

I went to Crowdin and the code seems to be ok:
- https://crowdin.com/translate/django-girls-tutorial/73/en-es#1249

**Comment by helenst** [comment]
So in English the markup looks like this:

```
Nice! See how the answer popped out? Python knows math! You could try other commands like:
- `4 * 5`
- `5 - 1`
- `40 / 2`
```
(https://github.com/DjangoGirls/tutorial/edit/master/en/python_introduction/README.md#L33)

but in Spanish (based on what Crowdin's done to it) it looks like this:
```
¡Bien! ¿Ves como salió la respuesta? ¡Python sabe matemáticas! Podrías intentar otros comandos como: - `4 * 5` - `5 - 1` - `40 / 2`
```
(https://github.com/DjangoGirls/tutorial/edit/master/es/python_introduction/README.md#L35)

It's similar in other non English languages. Crowdin's interpretation of that list is unlike other lists in that file - it seems to normally split out the items into individual translation units.

Could be because there's not a blank line before the list. I'm going to try adding one and see if that improves things. :)

**Comment by humitos** [comment]
People, I got frustrated on this (crowding has one old English version as base for translation and a lot of things are already fixed in the lastest English version -git master)

So, I'm sorry, but as we are having our 7th Django Girls Workshop tomorrow and I wanted to have a very nice and updated version, I decided to fork the project and mix all of these things / versions / collaborations and stuffs in just one place. I tried to communicate my frustration to some django girls emails, but it was taking too much time.

Finally, I wrote (mix all what we had, translate missing parts and put everything together in just one HTML page for tomorrow's workshop in Cuenca, Ecuador):

* http://argentinaenpython.com.ar/django-girls/tutorial/

You are free to share / use it and I will love if we can discuss the whole process of being a translator of the Django Girls tutorial (and other resources as well). We really need to improve it.

Thanks!
